docs/concepts/schedules.md

```diff
@@ -16,63 +16,48 @@ Schedules tell the Prefect API how to create new flow runs for you automatically
+To create a schedule from the UI, select Add. 
 
-<!--
-## Creating schedules
+![Prefect UI with Add button called out under Scheduling heading](../img/concepts/add-schedule-callout.png)
 
-You create a schedule by including a `schedule` parameter as part of the [deployment specification](/concepts/deployments/#deployment-specifications) for a deployment.
+Select Interval or Cron to create a schedule through the UI.
 
-First, import the schedule class that you want to use, then set the `schedule` parameter using an instance of that class.
+![Prefect UI with Interval button selected](../img/concepts/interval-schedule.png)
 
-```python
-from prefect.deployments import Deployment
-from prefect.orion.schemas.schedules import CronSchedule
+## Schedule types
 
-Deployment(
-    name="scheduled-deployment",
-    flow_location="/path/to/flow.py",
-    schedule=CronSchedule(cron="0 0 * * *"),
-)
-```
+Prefect supports several types of schedules that cover a wide range of use cases and offer a large degree of customization:
 
-If you change a schedule, previously scheduled flow runs that have not started are removed, and new scheduled flow runs are created to reflect the new schedule.
+- [`Cron`](#cron) is most appropriate for users who are already familiar with `cron` from previous use.
+- [`Interval`](#interval) is best suited for deployments that need to run at some consistent cadence that isn't related to absolute time.
+- [`RRule`](#rrules) is best suited for deployments that rely on calendar logic for simple recurring schedules, irregular intervals, exclusions, or day-of-month adjustments.
 
-To remove all scheduled runs for a flow deployment, update the deployment with no `schedule` parameter.
 
-## CronSchedule
+## Creating schedules through the deployment YAML file
 
-A `CronSchedule` creates new flow runs according to a provided [`cron`](https://en.wikipedia.org/wiki/Cron) string. Users may also provide a timezone to enforce DST behaviors.
+You create a schedule by including a `schedule` section as part of the [deployment YAML file](/concepts/deployments/#deploymentyaml) for a deployment. Here's an example that uses cron and specifies a timezone. 
 
-`CronSchedule` uses [`croniter`](https://github.com/kiorky/croniter) to specify datetime iteration with a `cron`-like format.
+```yaml
+schedule:
+  cron: 0 0 * * *
+  timezone: America/Chicago
+```
 
+This schedule will set flow runs in a deployment every day at midnight Central Time (CT).
 
-```python
-from prefect.deployments import Deployment
-from prefect.orion.schemas.schedules import CronSchedule
+## Cron
 
-Deployment(
-    name="cron-schedule-deployment",
-    flow_location="/path/to/flow.py",
-    schedule=CronSchedule(
-        cron="0 0 * * *",
-        timezone="America/New_York"),
-)
-```
+A `CronSchedule` creates new flow runs according to a provided [`cron`](https://en.wikipedia.org/wiki/Cron) pattern. Users may also provide a timezone to enforce DST behaviors.
```

Comment:
Suggest linking to the API definition for each of these. It'll make clear the available options.

Reply:
I Is it cool to leave out the API definition for now @tpdorsey ? I feel like it's a little confusing. Thinking we add back in with any Deployment updates. I removed mention of the Python classes in the changes I just pushed now.

Reply:
yep let's revisit later